test/visibility/import/relative/import-relative-error6.good

```diff
@@ -0,0 +1 @@
+import-relative-error6.chpl:4: error: 'this.' can only be used as prefix of import
```

Comment:
Yup, that's what I was worried about.  This is complaining about the `import super` statement, but is talking about `this.` in the error message.  That seems wrong to me